armada/handlers/armada.py

```diff
@@ -66,8 +66,8 @@ def __init__(self,
         self.wait = wait
         self.timeout = timeout
         self.tiller = Tiller(tiller_host=tiller_host, tiller_port=tiller_port)
-        self.documents = yaml.safe_load_all(file)
-        self.config = self.get_armada_manifest()
+        self.documents = list(yaml.safe_load_all(file))
```

Comment:
ah nevermind i misunderstood how the list was working but `apply` doesn't work with this change